### DISCUSSION TITLE
CodeHike and Details tag

### DISCUSSION THREAD

**Discussion by rabbitxtech** [code-hike/codehike#340] [Q&A]
When i nest code blocks into details tag, it doesn't seem to work on firefox, safari but still works on chrome, 
On chrome when i add 

.markdown-body details:not([open]) > *:not(summary) {
  display: none !important;
}

into css file, it's also not working

how can i fix this problem ?

![image](https://user-images.githubusercontent.com/127978275/227705406-1eccefb6-8cd3-43cc-9904-e6d3f8b9ee31.png)
![image](https://user-images.githubusercontent.com/127978275/227705415-c5662183-6b78-4fdf-8032-a0c0caa9a98f.png)
![image](https://user-images.githubusercontent.com/127978275/227706344-9fb0bc99-e44f-48ea-a98b-d15c27397464.png)

**Comment by pomber** [comment]
See #372
